integrations-and-supported-tools/pytorch/scripts/run_examples.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+pip install -r requirements.txt
+python Neptune_Pytorch_Support.py
+python Neptune_Pytorch_Support_Advanced.py
```

Comment:
I'd change the name to `Neptune_PyTorch_support_more_options.py`

Reply:
Done.